<?xml version="1.0"?>
<launch>

    <include file="$(find probot_gazebo)/launch/probot_anno_with_camera_vs/probot_anno_with_camera_vs_bringup_moveit.launch" />

    <!-- <node pkg="nn_vs" name="image_listenner" type="image_listenner.py" output="screen" /> -->

    <!-- <include file="$(find realsense2_camera)/launch/rs_camera.launch">
        <arg name="color_width"         value="1280"/>
        <arg name="color_height"        value="720"/>
        <arg name="enable_color"        value="true"/>
    </include> -->

    <!-- <node pkg="probot_demo" name="move" type="move.py" output="screen" /> -->
    


</launch>